packages/node_modules/@ciscospark/internal-plugin-lyra/src/lyra.js

```diff
@@ -11,8 +11,19 @@ const Lyra = SparkPlugin.extend({
   children: {
     space: Space,
     device: Device
-  }
+  },
 
+  sendUltraSoundToken(token) {
```

Comment:
Please add doc block

Reply:
What's in the response body? I wonder if we can give the method a name gives a better idea why we're calling it instead of just what it's doing.

Reply:
I renamed to `getAdvertisedEndpoint` so it makes more sense hopefully :)
Also added @typedef so we know how the body looks like